webgpu.h

```diff
@@ -496,6 +507,16 @@ typedef enum WGPUStoreOp {
     WGPUStoreOp_Force32 = 0x7FFFFFFF
 } WGPUStoreOp WGPU_ENUM_ATTRIBUTE;
 
+typedef enum WGPUSurfaceGetCurrentTextureError {
+	WGPUSurfaceGetCurrentTextureError_NoError = 0x00000000,
```

Comment:
We currently have 1 other thing called "NoError" and 6 called "Success". Maybe we should standardize on "Success"? And "Status" instead of "Error"?

(Actually, I don't even know what the other NoError could possibly be used for...)

Reply:
> (Actually, I don't even know what the other NoError could possibly be used for...)

Never mind, it's used by PopErrorScope when there's no error in the scope. That name makes sense.

Reply:
> Maybe we should standardize on "Success"

:+1: 

>  "Status" instead of "Error"?

If we go for `Status` instead of `Error` then imo we should remove DeviceLost from here, apps should be able to catch it via DeviceLostCallback anyway.

Reply:
That's a good point, I wonder if we should also remove it from WGPUErrorType (PopErrorScope) and maybe the others? In line with https://github.com/gpuweb/gpuweb/issues/1629

Reply:
Meeting: Decided to change to `Status` and `Status_Error`, but keep DeviceLost for now. New issue about DeviceLost: #209

Reply:
ack